components/21-atoms/table/_table.scss

```diff
@@ -0,0 +1,105 @@
+div.responsive-table,
+div.table-list {
+  .list-description,
+  caption {
+    @include theme('color', 'color-tertiary--lighten-1', 'table-caption-color');
+
+    margin: .4rem 0;
+    font-size: .6rem;
+    text-align: left;
+  }
+}
+
+div.responsive-table {
+  @include tablet {
+    display: block;
+  }
+
+  display: none;
+  overflow-x: auto;
+
+  &:focus {
+    border: 2px dotted;
+  }
+
+  table {
+    caption-side: bottom;
```

Comment:
Hey, never seen that one before! :trophy:

Reply:
Me neither :P had to google it.